src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -239,17 +239,20 @@ && stringAlreadyInUseElsewhereInSource(
             validationErrors.add(ValidationError.CVR_OVERVOTE_UNEXPECTEDLY_DEFINED);
           }
         }
-        if (fieldIsDefinedButShouldNotBeForProvider(
-            source.getFirstVoteColumnIndex(),
-            "firstVoteColumnIndex",
-            provider,
-            source.getFilePath())) {
-          validationErrors.add(ValidationError.CVR_FIRST_VOTE_UNEXPECTEDLY_DEFINED);
-        }
 
-        if (fieldIsDefinedButShouldNotBeForProvider(
-            source.getFirstVoteRowIndex(), "firstVoteRowIndex", provider, source.getFilePath())) {
-          validationErrors.add(ValidationError.CVR_FIRST_VOTE_ROW_UNEXPECTEDLY_DEFINED);
+        if (provider != Provider.CSV) {
+          if (fieldIsDefinedButShouldNotBeForProvider(
+              source.getFirstVoteColumnIndex(),
+              "firstVoteColumnIndex",
+              provider,
+              source.getFilePath())) {
+            validationErrors.add(ValidationError.CVR_FIRST_VOTE_UNEXPECTEDLY_DEFINED);
+          }
+
+          if (fieldIsDefinedButShouldNotBeForProvider(
+              source.getFirstVoteRowIndex(), "firstVoteRowIndex", provider, source.getFilePath())) {
+            validationErrors.add(ValidationError.CVR_FIRST_VOTE_ROW_UNEXPECTEDLY_DEFINED);
+          }
```

Comment:
Instead of modifying this section, can we just change the line earlier in this method from
```
} else if (provider == Provider.ESS) {
```
to
```
} else if (provider == Provider.ESS || provider == Provider.CSV) {
```
?

(This suggestion is related to my comments about shared functionality between ESS and CSV.)

Reply:
That would add a bunch of invalid checks for CSV (firstVoteColumnIndex, firstVoteRowIndex, idColumnIndex, precinctColumnIndex).

Note that the only change here is REMOVING this existing check from CSVs. If you click "Hide Whitespace" on github, that becomes more clear.

Reply:
But shouldn't CSV have those checks? Are we validating `firstVoteColumnIndex` and `firstVoteRowIndex` for CSV somewhere else?

(Same for `idColumnIndex` and `precinctColumnIndex` if you add support for those.)

Reply:
Ah, you're right -- those checks should be in here.

It would be nice to automate this a little more: the same information needs to be synced here, in `GuiConfigController.java`, and in the `Reader` constructor. I think we can consolidate a bit -- a task for later! For now, I'll add the checks here.

Reply:
I added a ticket to address this, since with the addition of the CSV checks, the function gets pretty hairy https://github.com/BrightSpots/rcv/issues/667